fabric_generator/file_parser.py

```diff
@@ -245,7 +95,12 @@ def parseFabricCSV(fileName: str) -> Fabric:
         i = [j for j in i if j != ""]
         if not i:
             continue
-        if i[0].startswith("ConfigBitMode"):
+        if i[0].startswith("Tile"):
+            parseTile(os.path.abspath(os.path.join(filePath, i[1])), tileTypes, tileDefs, commonWirePair)
```

Comment:
Rather than passing the object and appending the item to them, returning the object is preferred. Returning new objects makes the code more readable.

Reply:
Done, but what should I do with `commonWirePair`? Keep it as it is or return a tuple?

Reply:
We can do the same. Return as a tuple, then unpack directly. Something like `new_tiles, commonWirePair = parseTiles(os.path.abspath(os.path.join(filePath, i[1])))`

Reply:
Done